meta-mender-core/classes/mender-setup-image.inc

```diff
@@ -44,6 +44,18 @@ mender_update_fstab_file() {
 
     mkdir -p ${IMAGE_ROOTFS}/data
     printf "%-20s %-20s %-10s %-21s %-2s %s\n" ${tmpDataPart} /data ${MENDER_DATA_PART_FSTYPE} ${MENDER_DATA_PART_FSTAB_OPTS} 0 0 >> ${IMAGE_ROOTFS}${sysconfdir}/fstab
+
+    # TODO: skip if mender-partuuid is used
+    if [ -n "${MENDER_EXTRA_PARTS}" ]; then
```

Comment:
Can you add a test for this and error out with `bbfatal` if both are set?

Reply:
OK will do. Thanks.

Reply:
Also for comment alignment something like that:
```
diff --git a/meta-mender-core/classes/mender-part-images.bbclass b/meta-mender-core/classes/mender-part-images.bbclass
index 964b14f..ee10283 100644
--- a/meta-mender-core/classes/mender-part-images.bbclass
+++ b/meta-mender-core/classes/mender-part-images.bbclass
@@ -139,9 +139,9 @@ EOF
     cat >> "$wks" <<EOF
 part --source rawcopy --sourceparams="file=${IMGDEPLOYDIR}/${IMAGE_LINK_NAME}.dataimg" --ondisk "$ondisk_dev" --align $alignment_kb --fixed-size ${MENDER_DATA_PART_SIZE_MB} --mkfs-extraopts='${MENDER_DATA_PART_FSOPTS}' $part_type_params
 EOF
-# added extra partitions if exists
+    # added extra partitions if exists
     cat >> "$wks" <<EOF
-    ${@get_extra_parts_wks(d)}
+${@get_extra_parts_wks(d)}
 EOF
 
     cat >> "$wks" <<EOF

```

Thanks.

Reply:
Correct!